src/pharmpy/model/statements.py

```diff
@@ -1318,7 +1384,7 @@ class Compartment:
     ----------
     name : str
         Compartment name
-    dose : Dose
+    dose : Dose, tuple(Dose)
```

Comment:
Don't allow `Dose` here. The standard constructor should only take the type of the attribute.